todo_app/data/trello_items.py

```diff
@@ -0,0 +1,45 @@
+from flask import session
+from todo_app.data.item import Item
+from todo_app.data.trello_api_comms import api_request_get, api_request_post, api_request_put
+
+task_list = []
+
+def process_trello_records():
+    
+    items_dict = api_request_get()
+    
+    for card in items_dict:
+        duplicate_task = False
+        for task in task_list:
+            if (card['id'] == task.id):
+                duplicate_task = True
+        if (duplicate_task == False):        
+            new_task = Item.from_trello_cards(card)
+            task_list.append(new_task)            
```

Comment:
This "forgets" completed items when you restart the app. Can you instead change it to fetch items from both the "OPEN" and "CLOSED" lists?

The open ones should have status "To Do" as they do now, and the closed ones status "complete".